fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java

```diff
@@ -103,7 +103,6 @@ public Message getMessage(final FedoraEvent event,
         message.setStringProperty(BASE_URL_HEADER_NAME, baseURL);
         message.setStringProperty(USER_HEADER_NAME, event.getUserID());
         message.setStringProperty(USER_AGENT_HEADER_NAME, userAgent);
-        message.setStringProperty(PROPERTIES_HEADER_NAME, String.join(",", event.getProperties()));
```

Comment:
Should we also remove the declaration of `PROPERTIES_HEADER_NAME` above? I suspect it is no longer used.